gadfly/gap_fill_generator.py

```diff
@@ -30,15 +30,28 @@ def find_named_entities(self):
         return entities
 
     def summarize_sentences(self):
-        fs = FrequencySummarizer()
-        sents = []
-        for span in self._parsed_text.sents:
-            sent = [self._parsed_text[i] for i in range(span.start, span.end)]
-            tokens = []
-            for token in sent:
-                tokens.append(token.text)
-            sents.append(tokens)
-        sentences = fs.summarize(sents, 5)
+        option = 1
```

Comment:
Probably the best way to solve this is to use the strategy pattern, which you can see explained very well here: https://www.youtube.com/watch?v=-NCgRD9-C6o&index=3&list=PLF206E906175C7E07.

Here is the python equivalent of that here: https://github.com/faif/python-patterns/blob/master/strategy.py